packages/replay/src/util/isBrowser.ts

```diff
@@ -4,3 +4,9 @@ export function isBrowser(): boolean {
   // eslint-disable-next-line no-restricted-globals
   return typeof window !== 'undefined' && !isNodeEnv();
 }
+
+type ElectronProcess = { type?: string };
```

Comment:
Can we just combine this into the `isBrowser` check, to save a few bytes? So like here: https://github.com/getsentry/sentry-javascript/pull/6653/files#diff-4fcb105c4f00a77752d73d1f2bd5ba142aae2ec9e5b06290c39d6e7e27a29f75R5